### PR TITLE
Support new aws regions

### PR DESCRIPTION
**Pull request by ayalganem**
- support new two aws regions: `Bahrain (me-south-1)` and `Hong Kong (ap-east-1)`

### CODE REVIEW

**Comment by DanielRis** [comment]
@ayalganem thank you for picking this up so quickly. Any idea when a new version will be released? I try to determine if I should compile it by myself or should just wait a few hours until it has been released :)

File: dome9/common/providerconst/const.go [hunk 1]
```diff
@@ -77,7 +79,7 @@ var AzureSecurityGroupProtocol = []string{"UDP", "TCP", "ANY"}
 var AzureSecurityGroupSourceScopeTypes = []string{"CIDR", "IPList", "Tag"}
 
 // The 16 regions Dome9 manages in AWS cloud account
```
Comment (erezweinstein5-zz) [inline dome9/common/providerconst/const.go]: still 16?

**Review by erezweinstein5-zz — CHANGES_REQUESTED** [review]

**Comment by ayalganem** [comment]
@DanielRis Hi, it will be released tomorrow.

**Review by erezweinstein5-zz — APPROVED** [review]

**Comment by ayalganem** [comment]
@DanielRis Released :)

**Pull request merged**